lib/srv/termmanager.go

```
@@ -65,14 +65,14 @@ func NewTermManager() *TermManager {
 	return &TermManager{
 		writers:           make(map[string]io.Writer),
 		readerState:       make(map[string]bool),
-		closed:            false,
+		closed:            make(chan struct{}),
 		readStateUpdate:   sync.NewCond(&sync.Mutex{}),
 		incoming:          make(chan []byte, 100),
 		terminateNotifier: make(chan struct{}, 1),
 	}
 }
 
-func (g *TermManager) writeToClients(p []byte) int {
+func (g *TermManager) writeToClients(p []byte) (int, error) {
```

Do we need to return an error here if it's never assigned to?